measurement-tool.html

```diff
@@ -242,6 +243,246 @@
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">Right-click on a ROI and select <span class="bold">Add to Folder</span> to add the ROI to an existing or new folder.</p>
+
+<p>On opening, the <span class="bold">Add to Folder</span> dialog will only show folders currently displayed in the Management Tab.</p>
+
+<p>Use the <span class="bold">Filter ROI Folders</span> text box to filter by folder name.</p>
+
+<p>Click on the <span class="bold">Source</span> drop-down and select the <span class="bold">All Folders</span> option to display all the folders in the system.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/measurementToolFolderAddTo.jpg" class="align-center" src="images/measurementToolFolderAddTo.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<li><p class="first">Drag and drop a ROI onto a folder to add it, or out of a folder to remove it.</p>
+
+<p>Similarly folders can be dragged and dropped in and out of other folders.</p>
+
+<p>When dragging a ROI out of a folder to orphan it, it needs to be dragged clear of the last ROI at the bottom of the list.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<li><p class="first">Right-click on a ROI within a folder to and select <span class="bold">Remove from Folder</span> to orphan a ROI.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/measurementToolFolderRemove.jpg" class="align-center" src="images/measurementToolFolderRemove.jpg" style="width: 65%;" />
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">Right-click on a folder within another folder and choose <span class="bold">Move Folder</span> from the menu to move a folder and its contents to another folder or to orphan it.</p>
+
+<p>Select a folder or <span class="bold">None</span> in the Destination Folder window.</p>
+
+<p>Click<span class="bold">OK</span>.</p>
+
+<p>Click on the <span class="bold">Source</span> drop-down and select the <span class="bold">All Folders</span> option to display all the folders in the system.</p>
```

Comment:
Would it make more sense to move this line after the screenshot since that doesn't show this final point?